docs/cli-reference/README.md

```diff
@@ -2,20 +2,25 @@
 
 The Ark client provides a CLI that allows you to initiate ad-hoc backups, scheduled backups, or restores.
 
-*The files in this directory enumerate each of the possible `ark` commands and their flags. Note that you can also find this info with the CLI itself, using the `--help` flag.*
+[The files in the CLI reference directory][1] in the repository enumerate each of the possible `ark` commands and their flags. 
+This information is available in the CLI, using the `--help` flag.
 
 ## Running the client
 
-While it is possible to build and run the `ark` executable yourself, it is recommended to use the containerized version. Use the alias described in the quickstart:
+While it is possible to build and run the `ark` executable yourself, we recommend that you run the containerized version. 
```

Comment:
Please remove all text related to running the containerized version of the client. We only recommend they download a pre-built release, or if they desire, one they built from source.